src/buffer/out/TextAttribute.cpp

```diff
@@ -288,6 +288,11 @@ bool TextAttribute::IsItalic() const noexcept
     return WI_IsFlagSet(_attrs, CharacterAttributes::Italics);
 }
 
+bool TextAttribute::IsBold() const noexcept
+{
+    return WI_IsFlagSet(_attrs, CharacterAttributes::Bold);
+}

```

Comment:
This addition is unnecessary. The "bold" state can already be determined by the `IsIntense` method. But in terms of what we put in the HTML output, it should probably only be triggering a bold font if the `IntenseIsBold` mode is also set.